src/utilities/link/context.tsx

```diff
@@ -0,0 +1,4 @@
+import {createContext} from 'react';
+import {Link} from './link';
+
+export const LinkContext = createContext<Link>(new Link());
```

Comment:
You don't need the generic in the <> because TS is already smart enough to work that out based on your argument type